### PR TITLE
Feat: Label updates for Part 3 Awards, File Submissions - 2491 2492

### PR DESCRIPTION
**Pull request by AlexZorkin**

### CODE REVIEW

File: frontend/src/credit_transfers/CreditTransferAddContainer.js [hunk 1]
```diff
@@ -317,7 +317,7 @@ class CreditTransferAddContainer extends Component {
         isCommenting={this.state.isCommenting}
         isCreatingPrivilegedComment={this.state.isCreatingPrivilegedComment}
         key="creditTransferForm"
-        title="New Part 3 Award"
+        title="New Initiative Agreement Issuance"
         validationErrors={this.state.validationErrors}
```
Comment (prv-proton) [inline frontend/src/credit_transfers/CreditTransferAddContainer.js:321]: changes to confirmation Modal

**Comment by prv-proton** [comment]
If we try to open the file submissions, the title and attachment type still shows as P3A. Do we need a separate ticket for this?
image

**Comment by AlexZorkin** [comment]
> If we try to open the file submissions, the title and attachment type still shows as P3A. Do we need a separate ticket for this? image

I will followup with Justin on this one

**Comment by justin-lepitzki** [comment]
@prv-proton @AlexZorkin Good catch on the confirmation page Prashanth. Can we change that sentence to say "Are you sure you want to recommend issuance of compliance units for this Initiative Agreement?" and then have the buttons be Cancel (formerly No) and Recommend issuance (formerly Yes). 

For viewing a File Submission, I have created a separate ticket to change those labels (#2494). There are a few in the coming soon column that cover the label changes for file submissions.

**Pull request merged**